docs/release_notes/v0.11.0.md

```diff
@@ -0,0 +1,531 @@
+  
+# Dapr 0.11.0
+
+We're happy to announce the release of Dapr v0.11.0!
+
+The Dapr releases are increasingly focusing on operational concerns like stability, reliability, and upgradability in production environments.
```

Comment:
Consider this: 

> Our Dapr releases are increasingly focused on addressing operational concerns like: security, stability, and reliability. Additionally, given the growing number of Dapr deployments, we have further refined out upgrade process to enable durable production environments. 

Direct user feedback is the primary input into our release planning process. As with every release, we look forward to hearing from you on any friction you may have experienced using Dapr.